tests/common/Keystore/StoredKeyTests.cpp

```diff
@@ -272,27 +272,27 @@ TEST(StoredKey, WalletInvalid) {
 }
 
 TEST(StoredKey, LoadNonexistent) {
-    ASSERT_THROW(StoredKey::load(TESTS_ROOT + "/Keystore/Data/nonexistent.json"), invalid_argument);
+    ASSERT_THROW(StoredKey::load(TESTS_ROOT + "/common/Keystore/Data/nonexistent.json"), invalid_argument);
 }
 
 TEST(StoredKey, LoadLegacyPrivateKey) {
-    const auto key = StoredKey::load(TESTS_ROOT + "/Keystore/Data/legacy-private-key.json");
+    const auto key = StoredKey::load(TESTS_ROOT + "/common/Keystore/Data/legacy-private-key.json");
     EXPECT_EQ(key.type, StoredKeyType::privateKey);
     EXPECT_EQ(key.id, "3051ca7d-3d36-4a4a-acc2-09e9083732b0");
     EXPECT_EQ(key.accounts[0].coin, TWCoinTypeEthereum);
     EXPECT_EQ(hex(key.payload.decrypt(TW::data("testpassword"))), "7a28b5ba57c53603b0b07b56bba752f7784bf506fa95edc395f5cf6c7514fe9d");
 }
 
 TEST(StoredKey, LoadLivepeerKey) {
-    const auto key = StoredKey::load(TESTS_ROOT + "/Keystore/Data/livepeer.json");
+    const auto key = StoredKey::load(TESTS_ROOT + "/common/Keystore/Data/livepeer.json");
     EXPECT_EQ(key.type, StoredKeyType::privateKey);
     EXPECT_EQ(key.id, "70ea3601-ee21-4e94-a7e4-66255a987d22");
     EXPECT_EQ(key.accounts[0].coin, TWCoinTypeEthereum);
     EXPECT_EQ(hex(key.payload.decrypt(TW::data("Radchenko"))), "09b4379d9a41a71d94ee36357bccb4d77b45e7fd9307e2c0f673dd54c0558c73");
 }
 
 TEST(StoredKey, LoadPBKDF2Key) {
-    const auto key = StoredKey::load(TESTS_ROOT + "/Keystore/Data/pbkdf2.json");
+    const auto key = StoredKey::load(TESTS_ROOT + "/common/Keystore/Data/pbkdf2.json");
```

Comment:
How about having those common path in the same std::filesystem::path global variable in order to simplify future refactoring? https://en.cppreference.com/w/cpp/filesystem/path